### ISSUE TITLE
[Fix] Signals : Names are incorrect 

### ISSUE THREAD

**Issue by qk7b** [xee-lab/xee-sdk-android#13]
In the API the names of beams signals are not plurals.

Means that in the [sdk-api signals file](https://github.com/xee-lab/xee-sdk-android/blob/master/sdk-api/src/main/java/com/xee/api/entity/Signal.java) 

``` java
HIGH_BEAMS("HighBeamsSts"),
LOW_BEAMS("LowBeamsSts"),
HEAD_LIGHTS("HeadLightsSts"),
```

Should be replaced by

``` java
HIGH_BEAMS("HighBeamSts"),
LOW_BEAMS("LowBeamSts"),
HEAD_LIGHTS("HeadLightSts"),
```

**Comment by jcholin** [comment]
Closed by #17

**Comment by jcholin** [comment]
Closed by #17
